src/graph/visitor/ExtractFilterExprVisitor.cpp

```diff
@@ -276,6 +276,7 @@ bool ExtractFilterExprVisitor::visitLogicalOr(LogicalExpression *expr) {
         if (canNotPushedIndex != -1 || !canBeSplit) {
           return false;
         }
+        canBePushed_ = true;  // reset when the LogicalAnd can be split
```

Comment:
IMHO, this is not a good fix. Just fixed code path, but it's not a natural code logic.

Reply:
That's only what I can do.